subsys/bluetooth/services/fast_pair/fp_auth.c

```diff
@@ -245,6 +212,20 @@ int fp_auth_start(struct bt_conn *conn, bool send_pairing_req)
 	bt_auth_peer_passkeys[conn_idx] = EMPTY_PASSKEY;
 	update_conn_handling(conn, true);
 
+	err = bt_conn_auth_cb_overlay(conn, &conn_auth_callbacks);
+	if (err) {
+		return err;
+	}
+
+	if (!auth_info_cb_registered) {
+		err = bt_conn_auth_info_cb_register(&conn_auth_info_callbacks);
+		if (err) {
+			return err;
+		}
+
+		auth_info_cb_registered = true;
+	}
```

Comment:
Consider registering this using SYS_INIT. This way, you don't have to use static variable (and increase the number of state variables)

Reply:
sure